ctwrap/modules/adiabatic_flame.py

```diff
@@ -114,19 +114,27 @@ def run(name, chemistry=None, upstream=None, domain=None, loglevel=0):
     return out
 
 
-def save(filename, data, task=None):
+def save(filename, data, task=None,  groups=None, **kwargs):
     """
     This function saves the output from the run method
 
     Arguments:
         filename (str): naming of file
         data (Dict): data to be saved
         task (str): name of task if running variations
+        groups (list): group names if file exists
+        kwargs (dict): output information
     """
+    mode = "a"
+
+    force = kwargs.get('force_overwrite', False)
+    species = kwargs.get('species', 'X')
```

Comment:
For `kwargs`, separate between arguments that can be passed through directly to `write_hdf` (which should remain `kwargs`) and ones that are not (they should be named).

Reply:
As a minor comment, you can use `kwargs.pop` also to pick off individual items.